conf/defaults.ini

```diff
@@ -769,6 +769,7 @@ enable_alpha = false
 app_tls_skip_verify_insecure = false
 # Enter a comma-separated list of plugin identifiers to identify plugins that are allowed to be loaded even if they lack a valid signature.
 allow_loading_unsigned_plugins =
+marketplace_url = https://grafana.com/grafana/plugins/
```

Comment:
You need to update conf/sample.ini also. And maybe documentation also (unsure)?

Reply:
This is the same value that was hardcoded on front end before, so need to check if it needs to be documented.

Reply:
It sounds to me as if it should be documented, since it's a new configuration parameter.

Reply:
I think we should document it. There's not really anything to lose from documenting it but not doing so might be confusing.